### DISCUSSION TITLE
如何指定simbot的登录设备

### DISCUSSION THREAD

**Discussion by luoyuemin** [simple-robot/Discussions#139] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

可以切换为手表登录嘛现在

**Comment by ForteScarlet** [comment]
参考组件文档 [BOT配置文件](https://component-mirai.simbot.forte.love/docs/bot-config) 中的属性 `config.protocol`
